### PR TITLE
fix: vault secrets webhook namespace selector

### PR DESCRIPTION
**Pull request by j-zimnowoda**
Fixes: #356

### CODE REVIEW

File: values/vault-secrets-webhook/vault-secrets-webhook.gotmpl [hunk 1]
```diff
@@ -1,22 +1,13 @@
...
-        - keycloak
-        - knative-serving
-        - gatekeeper-system
-        - istio-operator
-        - monitoring
-        - maintenance
-        - overprovisioner
-  # matchLabels:
-  #   vault-injection: enabled
+        {{- range $v.k8s.namespaces }}
```
Comment (Morriz) [inline values/vault-secrets-webhook/vault-secrets-webhook.gotmpl:9]: can we just make it opt in? We only offer vault to `team-*` namespaces so lets only capture those.

Comment (Morriz) [reply values/vault-secrets-webhook/vault-secrets-webhook.gotmpl:9]: and are we using this from vault? I thought all secrets were coming from external-secrets...if not we can just skip this whole part, right?

**Review by Morriz — CHANGES_REQUESTED** [review]

File: values/vault-secrets-webhook/vault-secrets-webhook.gotmpl (hunk 1 above)
Comment (j-zimnowoda) [reply values/vault-secrets-webhook/vault-secrets-webhook.gotmpl:9]: vault-secrets-webhook is not used right now. It is an option that can be enabled.

Comment (Morriz) [reply values/vault-secrets-webhook/vault-secrets-webhook.gotmpl:9]: By design we don't use it directly, so lets remove it

**Review by Morriz — CHANGES_REQUESTED** [review]
after removal of the vault injection namespace selector I think we are ready

File: values/vault-secrets-webhook/vault-secrets-webhook.gotmpl (hunk 1 above)
Comment (j-zimnowoda) [reply values/vault-secrets-webhook/vault-secrets-webhook.gotmpl:9]: ok

Comment (j-zimnowoda) [reply values/vault-secrets-webhook/vault-secrets-webhook.gotmpl]: done

**Review by Morriz — APPROVED** [review]

**Review by svatwork — APPROVED** [review]

**Pull request merged**